### PR TITLE
Update FLA.md

### PR DESCRIPTION
**Pull request by magdawalkiewicz25**

### CODE REVIEW

File: FLA.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+#### Babbling
+Between the ages of six and eight months, the kid is sitting up and generating a variety of vowels and consonants. **Babbling** is a term used to characterize this sort of sound creation. Because adults react to babble as if it were the kid's contribution to social engagement, this **"pre-language" use of sound** gives the infant some experience with the  social role of speech.
+#### The One-Word Stage 
+Children begin to make a range of identifiable single-unit utterances **between the ages of twelve and eighteen months**. The one-word stage, as it is known, is characterized by speech in which single phrases are employed for things. The term **holophrastic speech** refers to an utterance that can be a word, a phrase, or a sentence. Although, holographic expressions seem to be used to name objects, they may also appear in situations that indicate that children are already expanding their use.
```
Comment (dsolska) [inline FLA.md]: I'd remove the comma after 'although' :)

File: FLA.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+#### The Two-Word Stage
+The two-word stage can begin as early as **eighteen to twenty months**, depending on what we consider an incidence of two different words used simultaneously. Regardless of what the child is actually trying to communicate through this form of expression, the important functional consequence is that adults or older children behave as if they are communicating. In other words, the kid not only makes speech, but also receives feedback indicating that the utterance was successful as a contribution to the engagement.
+#### Telegraphic Speech
+Between the ages of two and two and a half, the kid begins to produce a considerable number of utterances that may be classed as **"multiple-word"** speech. At this stage, the kid has obviously established some sentence-building skills and can correctly organize words. By the age of two and a half, the child's language has grown significantly, and he or she can initate more conversations. At this stage, it is important to evaluate what type of effect adults have on the development of the child's speech.
```
Comment (dsolska) [inline FLA.md]: Spelling mistake in 'initiate' (missing 'i' after 't')

File: FLA.md [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+#### Caregiver speech
+A conversational framework that appears to assign an interactive role to the young kid even before he or she becomes a verbal participant is built into a lot of caregiver speech. This sort of structure becomes obvious when we look at an excerpt from a mother's speech to her child as if it were a two-party discussion. If the kid is developing a system for combining sounds and words, the simpler models created by the engaging adult may provide **useful insights** to the basic structural organization involved.
```
Comment (dsolska) [inline FLA.md]: I'd correct 'to' - 'provide useful insights **into**'

**Review by dsolska — COMMENTED** [review]
Just a few comments, looks really good! :)

File: FLA.md [hunk 4]
```diff
@@ -1,4 +1,23 @@
 # First Language Acquisition
+### The Acquisition Schedule 
+A child's broad linguistic capacity must be brought into contact with a specific language such as English throughout the first two or three years of growth. What is also important is the significance of **cultural transmission**, which means that a child's language skills are acquired in a specific language-using context rather than being genetically inherited. A child must be able to hear a language in order to speak it.
```
Comment (asiamac) [inline FLA.md]: Instead of `in order to` maybe `to` would be enough?

File: FLA.md [hunk 5]
```diff
@@ -1,4 +1,23 @@
...
+#### Cooing
+**Cooing** has been described as the earliest use of speech-like noises. The kid gradually gains the ability to produce sequences of vowel-like sounds, particularly high vowels, throughout the first few months of life.
+By four months of age, the child's growing ability to bring the back of the tongue into consistent contact with the back of the palate allows the infant to produce sounds comparable to velar consonants and, therefore the frequent description of "cooing" or "gooing" for this sort of output. According to research, infants can hear the difference between vowels and differentiate between syllables **by the age of five months**.
+#### Babbling
+Between the ages of six and eight months, the kid is sitting up and generating a variety of vowels and consonants. **Babbling** is a term used to characterize this sort of sound creation. Because adults react to babble as if it were the kid's contribution to social engagement, this **"pre-language" use of sound** gives the infant some experience with the  social role of speech.
```
Comment (asiamac) [inline FLA.md]: Double space between 'the' and 'social'.

**Review by asiamac — APPROVED** [review]

**Pull request merged**